ppml/services/ehsm/kubernetes/README.md

```diff
@@ -22,48 +22,47 @@
 - Please make sure you have a usable https proxy.
 - Please make sure your **CPU** is able to run PCCS service, which generate and verify quotes.
 - Please make sure you have a reachable **NFS**.
-- Please make sure you have already installed **[helm](https://helm.sh/)**.
 - Please make sure you have an usable PCCS ApiKey for your platform. The PCCS uses this API key to request collaterals from Intel's Provisioning Certificate Service. User needs to subscribe first to obtain an API key. For how to subscribe to Intel Provisioning Certificate Service and receive an API key, goto https://api.portal.trustedservices.intel.com/provisioning-certification and click on 'Subscribe'.
 
 
 
 Now you have already had a PCCS image.
 
 
-## 1. Deploy BigDL-PCCS on Kubernetes with Helm Charts
+## 1. Deploy BigDL-PCCS on Kubernetes
 If you already have a BidDL-PCCS service on Kubernetes, please skip this step.
 
-If not, please **[deploy BigDL-PCCS on Kubernetes with Helm Charts](https://github.com/intel-analytics/BigDL/tree/main/ppml/services/pccs/kubernetes)**.
+If not, please **[deploy BigDL-PCCS on Kubernetes](https://github.com/intel-analytics/BigDL/tree/main/ppml/services/pccs/kubernetes)**.
 ## 2. Start BigDL-eHSM-KMS on Kubernetes 
 
 Please make sure current workdir is `kubernetes`.
 
-Then modify parameters in `values.yaml` as following:
+Then modify parameters in `install-bigdl-ehsm-kms.sh` as following:
 
 ```shell
 # reset of other parameters in values.yaml is optional, please check according to your environment
-nfsServerIP: your_nfs_server_ip                   --->   <the_IP_address_of_your_NFS_server>
-nfsPath: a_nfs_shared_folder_path_on_the_server   --->   <an_existing_shared_folder_path_on_NFS_server>
+export nfsServerIp=your_nfs_server_ip                   --->   <the_IP_address_of_your_NFS_server>

```

Comment:
Should not add export here. Otherwise users may forget to modify the .sh script. Keep the old style.